src/main/resources/messages.properties

```diff
@@ -52,6 +52,7 @@ label.fillTheFields=Fill in the following fields:
 label.registration=Registration
 label.personal=Personal
 label.area=area
+label.customers=Customers
```

Comment:
Since we have i18n implemented, you should internationalise your JSPs, pay attention to possiible duplicates